packages/salesforcedx-vscode-lwc/snippets/lwc.json

```diff
@@ -0,0 +1,224 @@
+{
+  "lwc-event": {
+    "prefix": "lwc-event",
+    "description": "Creates a standard event without custom details",
+    "body": [
+      "const ${1:event} = new CustomEvent('${2:event-name}');",
+      "this.dispatchEvent(${1:event});"
+    ]
+  },
+  "lwc-event-data": {
+    "prefix": "lwc-event-details",
+    "description": "Creates a standard event for custom details",
+    "body": [
+      "const ${1:event} = new CustomEvent('${2:event-name}', {",
+      "    detail: { $0 }",
+      "});",
+      "this.dispatchEvent(${1:event});"
+    ]
+  },
+  "lwc-event-listener": {
+    "prefix": "lwc-event-listener",
+    "description": "Creates an event listener",
+    "body": [
+      "this.template.addEventListener('${1:event-name}', this.${2:local-method-name}.bind(this));"
+    ]
+  },
+  "lwc-wire-apex-property": {
+    "prefix": "lwc-wire-apex-property",
+    "description": "Creates a @wire'd property for Apex",
+    "body": [
+      "@wire(${1:apex-method-name}, {$0})",
+      "${2:apex-property-name}"
+    ]
+  },
+  "lwc-wire-get-record-property": {
+    "prefix": "lwc-wire-get-record-property",
+    "description": "Creates a @wire'd property for getRecord",
+    "body": [
+      "@api recordId;",
+      "",
+      "@wire(getRecord, { recordId: '\\$recordId', fields: [ $0 ] } )",
+      "${2:record-property-name}"
+    ]
+  },
+  "lwc-wire-get-record-function": {
+    "prefix": "lwc-wire-get-record-function",
+    "description": "Creates a @wire'd function for getRecord",
+    "body": [
+      "@api recordId;",
+      "",
+      "@wire(getRecord, { recordId: '\\$recordId', fields: [ $0 ] } )",
+      "${2:record-function-name} ({error, data}) {",
+      "    if (error) {",
+      "        // TODO: Error handling",
+      "    } else if (data) {",
+      "        // TODO: Data handling",
+      "    }",
+      "}"
+    ]
+  },
+  "lwc-wire-apex-function": {
+    "prefix": "lwc-wire-apex-function",
+    "description": "Creates a @wire'd function for Apex with data modification",
+    "body": [
+      "@wire(${1:apex-method-name}, {$0})",
+      "apex${2:apex-function-name} ({error, data}) {",
```

Comment:
Hi @muenzpraeger 
I believe this is a typo -- there is an extra "apex" at the beginning of the line. 

Also, are `apex-function-name` and `apex-method-name` referring to the same name? If so, I think it would be better to choose one to use consistently.

Reply:
Methods exist in Apex, functions in JavaScript. Hence the distinction.

And yes, I've chosen to prefix things with `apex` Not necessary, but helps to distinct.